### PR TITLE
(IMAGES-22) Windows 10 Template

### PR DESCRIPTION
**Pull request by jcoconnor**
Add the Windows 10 Packer Templates.

### CODE REVIEW

**Comment by jcoconnor** [comment]
Ping @puppetlabs/agent-team This PR is ready for review.

Have tested this against Fusion on Mac and it builds fine.
Next step is to deploy jobs to jenkins-imaging so that we can deploy a test machine to vmpooler.

File: scripts/windows/slipstream.package.ps1 [hunk 1]
```diff
@@ -23,7 +23,6 @@ reg.exe ADD "HKLM\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate\AU" /v "Sche
 reg.exe ADD "HKLM\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate\AU" /v "ScheduledInstallTime" /t REG_DWORD /d 3 /f
 reg.exe ADD "HKLM\SOFTWARE\Policies\Microsoft\Windows\WindowsUpdate\AU" /v "UseWUServer" /t REG_DWORD /d 1 /f
 
-
```
Comment (glennsarti) [inline scripts/windows/slipstream.package.ps1]: nit: should probably be in it's own commit

Comment (jcoconnor) [inline scripts/windows/slipstream.package.ps1]: Yeah - will remove it altogether.
In the original commit I had modified this file (to fix a win-10 issue with the slipstream).
But when I found that there were issues with WSUS I just removed the slipstream bits completely, but left this whitespace change in accidentally.

Comment (jcoconnor) [inline scripts/windows/slipstream.package.ps1]: Done
